docs/overview.md

```diff
@@ -1,19 +1,19 @@
 # Kubernetes User Documentation
 
-The Kubernetes API currently manages 3 main resources:
-* [pods](pods.md)
-* [replication controllers](replication-controller.md)
-* [services](services.md)
+Kubernetes is an open source orchestration system for containers (currently Docker containers). It schedules containers onto nodes in a compute cluster and actively manages these containers to ensure that the state of the cluster continually matches the user's intentions.
```

Comment:
The 2nd sentence is pretty good. It touches upon key system characteristics, such as scheduling, multi-node clusters, and control loops driving towards desired state.

Characterizing Kubernetes as a container orchestration system does us a disservice, IMO, though we need to relate what it does to what users might be familiar with. And, if one wants to get picky, it's not strictly correct, in that the scheduler interaction is closer to choreography than orchestration.

I'll try to think of alternate wording as I read the rest.
The top-level README.md states (also with bad hyphenation):
  Kubernetes is an open source system for managing containerized applications across multiple hosts, providing basic mechanisms for deployment, maintenance, and scaling of applications.